core/common/src/main/java/alluxio/concurrent/ManagedBlockingUfsForwarder.java

```diff
@@ -483,30 +483,36 @@ public AlluxioURI resolveUri(AlluxioURI ufsBaseUri, String alluxioPath) {
 
   @Override
   public void setAclEntries(String path, List<AclEntry> aclEntries) throws IOException {
-    new ManagedBlockingUfsFunction() {
+    new ManagedBlockingUfsMethod<Void>() {
       @Override
-      public void execute() throws IOException {
+      public Void execute() throws IOException {
         mUfs.setAclEntries(path, aclEntries);
+        // For obeying the contract. Returned object not utilized.
```

Comment:
These comments are not necessary, the signature should be sufficient explanation.